.github/workflows/build-distribute.yml

```
@@ -0,0 +1,208 @@
+      - name: Verify version
+        run: |
+          if [ -z "$VERSION_NAME" ]; then 
+            echo "Version name could not be extracted. Exiting." exit 1 
+          else 
+            echo "Version name is $VERSION_NAME" 
+          fi
+
+      - name: Build and Distribute app to Firebase
+        id: build-and-distribute
+        run: |
+
```

actionlint (1.7.4)

parser did not reach end of input after parsing the expression. 2 remaining token(s) in the input: "IDENT", "IDENT"

(expression)